[FIX] sale_amazon: fix information displayed (#2366)
odoo/documentation: content/applications/sales/sales/amazon_connector/manage.rst

     name (e.g. create a carrier named `Colissimo`). This name is case insensitive, but be careful
     about typos, as Amazon won't recognize them.
   - Create a delivery carrier named `Self Delivery` to inform Amazon that you make your own
     deliveries. You still have to enter a tracking reference.

>    - Keep in mind that the customer is notified by email about the delivery, and the carrier as well
>      as the tracking reference are then displayed.

Reviewer: Small wording change for clarification.
```suggestion
   - Keep in mind that the customer is notified by email about the delivery, and the carrier and tracking reference are displayed in the email to the customer.
```
